.github/workflows/scripts/mesh_deploy.sh

```diff
@@ -0,0 +1,27 @@
+MESH=$1
+
+echo "checking if mesheryctl is installed"
+	
+if mesheryctl
+	
+then
+	echo found mesheryctl, deploying $MESH
+	mesheryctl mesh deploy $mesh
+	
+else
+	printf "Mesherycyl not found. \nInstalling...\n"
+	install_mesheryctl
+	echo "Installed mesheryctl successfully!"
+	mesheryctl mesh deploy $MESH
+	
+fi
+
+
+
+
+
+install_mesheryctl(){
+  curl -L https://github.com/meshery/meshery/releases/download/v0.5.67/mesheryctl_0.5.67_Linux_x86_64.zip -o mesheryctl.zip
+  unzip -n mesheryctl.zip 
+  mv mesheryctl /usr/local/bin/mesheryctl
```

Comment:
If I remember right, there would be a permission issue? Maybe there is other solution here?https://unix.stackexchange.com/questions/37724/permissions-ownership-of-usr-local-bin

Reply:
🤔 I'm not sure, as far as I know, workflow runners usually run as a privileged user

Reply:
Sounds good. We don't need to change this.

Reply:
If you use the approved installation procedure, this is already considered for.

// @hexxdump